include/variables.h

```diff
@@ -2817,10 +2817,10 @@ extern s16 D_80153960[];
 //extern ? D_80154AE8;
 //extern ? D_80154AFC;
 //extern ? D_80154B00;
-extern u8 gAudioBankTable[];
-extern u8 D_80155340[];
-extern u8 gSequenceTable[];
-extern u8 gAudioTable[];
+extern AudioBankTable gAudioBankTable;
+extern u16 D_80155340[];
+extern SequenceTable gSequenceTable;
+extern AudioTable gAudioTable;
```

Comment:
u8[] is more correct here: it's a blob of bytes that later gets interpreted as various audio tables. A cast at the usage site is a better fix. AudioBankTable etc. are dubious because they can't be initialized due to the trailing `[1]` mock flexible array member